components/engine_rocks/src/write_batch.rs

```diff
@@ -263,6 +271,10 @@ impl engine_traits::WriteBatch<RocksEngine> for RocksWriteBatchVec {
         }
         Err(Error::Engine("no save point".into()))
     }
+
+    fn merge(&mut self, _: &Self) {
+        panic!("merge is not implemented for RocksWriteBatchVec");
```

Comment:
This can be easily implemented. /cc @Little-Wallace

Reply:
I implemented it before https://github.com/gengliqi/tikv/blob/raftstore-async-io-tmp/components/engine_rocks/src/write_batch.rs#L174-L260. I think the change is not very small so I prefer to implement in later PR. Also, there is no side effect because the raft db can not use `RocksWriteBatchVec`.